service/project.go

```diff
@@ -373,13 +372,19 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 				uis.LoggedError(w, r, http.StatusInternalServerError, errors.Wrapf(err, "error checking for commit queue aliases"))
 				return
 			}
-			if len(aliases) == 0 {
+
+			remainingAliases := 0
+			for _, a := range aliases {
+				// only consider aliases that won't be deleted
+				if !util.StringSliceContains(responseRef.DeleteAliases, a.Alias) {
```

Comment:
I think this will return an error if the user deletes a single commit queue alias and there is more than one.

Reply:
No, that would be true if we returned immediately after finding one. We maintain `remainingAliases` and add to it every time we find a commit queue alias that won't be deleted, and we only return an error if this count is 0, i.e. there are no un-deleted commit queue aliases.

Reply:
I misunderstood: `responseRef.DeleteAliases` contains the ids of aliases we want to delete.
Maybe we want to check DeleteAliases doesn't contain `a.Id.Hex()`?

Reply:
Yeah I noticed this in staging tests whoops! It's fixed.